src/components/explore/ProbeExplorer.svelte

```diff
@@ -56,19 +56,71 @@
       ? normData.filter((d) => !isEmpty(d.non_norm_histogram))
       : normData;
 
-  function smoothenData(data, accessor, level) {
-    // Interpolates percentiles values by applying a Moving Average.
-    if (!level) {
-      return data;
+
+  function getInterpPercBtnRanksForHistogram(histogram, percentiles) {
+    // Compute cumulative frequencies
+    var cumFreq = [];
+    var totalFreq = 0;
+    for (var i = 0; i < histogram.length; i++) {
+      totalFreq += histogram[i].value;
+      cumFreq.push({ bin: histogram[i].bin, cumFreq: totalFreq });
+    }
+    const normalizer = cumFreq.at(-1).cumFreq
```

Comment:
This is just `totalFreq`, fwiw. Except in the edge case where `histogram` is empty, but then the `.cumFreq` would throw an error. (It's probably impossible?)